src/typeDefs/mutations.ts

```diff
@@ -67,6 +69,8 @@ export const mutations = gql`
 
     createPost(data: PostInput!, file: String): Post @auth
 
+    createTag(title: String!, parentTag: ID, organizationId: ID): Tag @auth
```

Comment:
Same as `toggleTagAssign`, make the input type, follow the naming convention.
Rename `title` to `name`, `parentTag` to `parentTagId`.